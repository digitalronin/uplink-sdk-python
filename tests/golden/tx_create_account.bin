00000000: 03 EE 00 80 61 66 30 62  33 33 64 39 65 38 61 66  ....af0b33d9e8af
00000010: 32 62 37 37 31 32 30 30  61 38 37 30 31 32 66 63  2b771200a87012fc
00000020: 36 61 61 30 37 62 33 39  31 64 38 61 63 30 31 62  6aa07b391d8ac01b
00000030: 34 30 39 37 35 33 35 39  61 37 64 33 35 64 33 38  40975359a7d35d38
00000040: 66 31 33 62 39 64 31 64  36 35 62 32 38 61 39 61  f13b9d1d65b28a9a
00000050: 31 61 34 31 33 34 33 36  63 66 36 64 63 35 62 61  1a413436cf6dc5ba
00000060: 66 63 62 35 35 31 61 34  39 35 33 62 38 66 33 30  fcb551a4953b8f30
00000070: 63 37 35 31 64 31 31 31  32 36 32 65 64 30 37 61  c751d111262ed07a
00000080: 31 33 66 64 00 03 47 4D  54 00 03 00 03 62 61 78  13fd..GMT....bax
00000090: 00 03 66 6F 6F 00 03 66  61 78 00 03 62 61 72 00  ..foo..fax..bar.
000000A0: 05 73 74 75 66 66 00 03  6B 65 79                 .stuff..key