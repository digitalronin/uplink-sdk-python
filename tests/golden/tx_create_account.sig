r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 25144991201232260910098675914000653886995007070611291546420545233304112119026
