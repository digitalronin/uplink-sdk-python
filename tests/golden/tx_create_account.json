{
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "header": {
        "tag": "TxAccount", 
        "contents": {
            "tag": "CreateAccount", 
            "contents": {
                "timezone": "GMT", 
                "pubKey": "af0b33d9e8af2b771200a87012fc6aa07b391d8ac01b40975359a7d35d38f13b9d1d65b28a9a1a413436cf6dc5bafcb551a4953b8f30c751d111262ed07a13fd", 
                "metadata": {
                    "fax": "bar", 
                    "stuff": "key", 
                    "bax": "foo"
                }
            }
        }
    }, 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE03NTU1NTczODcyNTA5NDkxMTUzODYzNTQ0NjM2ODQwNDA1ODEwNDIzMTMwODk5ODIwMTM2NjQ1NTAyNTU0NDg2MzY3MTkxOTAzMzM0Mw=="
}